pkg/nvmeclient/nvme_client.go

```diff
@@ -466,6 +471,7 @@ func Connect(request *ConnectRequest) (*CtrlIdentifier, error) {
 		}
 	}
 	request.Traddr = traddr
+	request.Hostid = uuid.NewMD5(uuid.Nil, []byte(request.Hostnqn)).String()
```

Comment:
@guy-lb I think that you should also do this in the Discover?

Reply:
@sagi-lb done